### PR TITLE
Update message.json

### PR DESCRIPTION
**Pull request by zVapor-Dev**
**Please describe the changes this PR makes and why it should be merged:**
I added the language: Dutch to the message.json file.

**Status and versioning classification:**

### CODE REVIEW

**Comment by xHyroM** [comment]
please change it to Garlic-Team/dev-latest branch

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:30]:
```suggestion
        "nederlands": "Je moet de {PERMISSION} permissie hebben om dit commando te gebruiken."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:45]:
```suggestion
        "nederlands": "De client moet de {PERMISSION} permissie hebben om dit commando uit te voeren."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:60]:
```suggestion
        "nederlands": "Je hebt geen van de vereiste rollen om dit commando te kunnen gebruiken! Je hebt een van de volgende rollen nodig: {ROLES}"
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:75]:
```suggestion
        "nederlands": "Je moet {COOLDOWN} wachten voor je dat commando opnieuw kan gebruiken."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:90]:
```suggestion
        "nederlands": "Het commando {COMMAND} is niet gevonden!"
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:105]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in text kanalen."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:120]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in nieuws kanalen."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:135]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in threads."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:150]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in nsfw kanalen."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:180]:
```suggestion
        "nederlands": "Je tijd om deze vraag te beantwoorden is op. Voer het commando opnieuw uit."
```

File: src/util/message.json
Comment (S222em) [inline src/util/message.json:240]:
```suggestion
        "nederlands": "Kies aub een van de volgende subcommando's: {choices}."
```

**Review by S222em — CHANGES_REQUESTED** [review]
Could you also change the branch to dev-latest?

**Comment by zVapor-Dev** [comment]
alr I updated the suggestions and changed the branch

**Review by S222em — APPROVED** [review]

**Comment by xHyroM** [comment]
Please resolve conflicts

**Comment by zVapor-Dev** [comment]
ok done

**Comment by S222em** [comment]
Something went wrong here. Make sure the branch you merge with is dev-latest and not the next branch

**Comment by zVapor-Dev** [comment]
> Something went wrong here. Make sure the branch you merge with is dev-latest and not the next branch

On my end it says awaiting approval

**Comment by xHyroM** [comment]
Don't edit package.json

**Comment by zVapor-Dev** [comment]
You asked me to resolve the conflicts so I did. Package.json was one of them

**Review by S222em — CHANGES_REQUESTED** [review]
This is a merge with the package.json from the work in progress next version of commands. We can not merge it in this state. Please change the package.json to its original state from the correct branch.

**Comment by zVapor-Dev** [comment]
But this is the one from gcommands dev

**Comment by S222em** [comment]
> But this is the one from gcommands dev

It's definitely not. This is the correct package.json https://github.com/Garlic-Team/GCommands/blob/latest-dev/package.json

**Comment by zVapor-Dev** [comment]
I’ll have a look when I’m home

**Comment by zVapor-Dev** [comment]
I should've added it now. please re-review

**Review by xhyrom — APPROVED** [review]

**Review by S222em — APPROVED** [review]

**Pull request merged**
